bin/cheritest/cheritest_bounds_globals.c

```diff
@@ -81,12 +81,14 @@
 	void								\
 	test_bounds_##test(const struct cheri_test *ctp __unused)	\
 	{								\
-		void * __capability allocation =				\
+		void * __capability allocation =			\
 		    (__cheri_tocap void * __capability )&test;		\
 		size_t allocation_offset = cheri_getoffset(allocation);	\
 		size_t allocation_len = cheri_getlen(allocation);	\
 		size_t pointer_offset = cheri_getoffset(test##p);	\
 		size_t pointer_len = cheri_getlen(test##p);		\
+		size_t size = sizeof(test);				\
+		size_t round_size = CHERI_REPRESENTABLE_LENGTH(size);	\
```

Comment:
I would call this rounded_size but probably not worth updating the pr for this.

Reply:
Eh, it's easy enough. :)